geode-core/src/main/java/org/apache/geode/internal/cache/OverflowOplog.java

```diff
@@ -724,8 +727,31 @@ public final void flush() throws IOException {
         if (bb != null && bb.position() != 0) {
           bb.flip();
           int flushed = 0;
+          int numChannelRetries = 0;
           do {
-            flushed += olf.channel.write(bb);
+            int channelBytesWritten = 0;
+            final int bbStartPos = bb.position();
+            final long channelStartPos = olf.channel.position();
+            // differentiate between bytes written on this channel.write() iteration and the
+            // total number of bytes written to the channel on this call
+            channelBytesWritten += olf.channel.write(bb);
```

Comment:
Instead of "+="; we could just assign the value...Its not really makes any difference; its just when you read this line, you don't have to know its previous value...